File: 백준/Bronze/2566. 최댓값/최댓값.py
arr=[list(map(int,input().split())) for i in range(9)]

x=0
y=0
mx=-1

for i in range(9):
    for j in range(9):
        if arr[i][j]>mx:
            mx=arr[i][j]
            x=i+1 #행열이 1부터 시작
            y=j+1

print(mx)
print(x,y)

File: 백준/Bronze/2566. 최댓값/최댓값.java
import java.io.*;
import java.util.StringTokenizer;


public class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int[][] arr=new int[9][9];
		int mx=Integer.MIN_VALUE;
		int idx_x=0;
		int idx_y=0;
		
		
		for (int i=0;i<9;i++) {
		String[] br1=br.readLine().split(" ");
		for (int j=0; j<9; j++) {
			 arr[i][j]=Integer.parseInt(br1[j]);
			 if (arr[i][j]>mx) {
				 mx=arr[i][j];
				 idx_x=i+1;
				 idx_y=j+1;
			 }
		}
		}
		System.out.println(mx);
		System.out.print(idx_x+" "+idx_y);
		}
			}